Grammar/python.gram

```diff
@@ -1396,6 +1396,7 @@ invalid_elif_stmt:
 invalid_else_stmt:
     | a='else' ':' NEWLINE !INDENT {
         RAISE_INDENTATION_ERROR("expected an indented block after 'else' statement on line %d", a->lineno) }
+    | 'else' ':' block 'elif' { RAISE_SYNTAX_ERROR("elif not allowed after else")}
```

Comment:
We can use "elif' block follows an 'else' block" to mimic the "positional patterns follow keyword patterns" error.

Reply:
If I didn't already know Python, I would find it unclear if "elif' block follows an 'else' block" is a statement about what the problem is or how it's supposed to be.

Reply:
Note that there will be "SyntaxError: elif block follows else block". The fact that there is the SyntaxError prefix would makes it clear to me that the message explains the issue. Otherwise I agree with you that the standalone message would be confusing.